ctapipe/containers.py

```diff
@@ -724,14 +735,23 @@ class ReconstructedContainer(Container):
     )
 
 
+class TelescopeReconstructedContainer(ReconstructedContainer):
```

Comment:
This removes energy, geometry and classification from the telescope reconstructed parameter, these fields will actually be filled for mono predictions in the ml module in #1767, so they should not be removed I think.

Reply:
Sorry, didn't see the inheritance, all good